config/environment.rb

```diff
@@ -3,3 +3,14 @@
 
 # Initialize the Rails application.
 MiqBot::Application.initialize!
+
+# Action
+ActionMailer::Base.smtp_settings = {
```

Comment:
Initializers live in config/initializers/ you can read more [here](http://guides.rubyonrails.org/configuring.html#using-initializer-files)